etl/src/main/java/org/pih/hivmigration/etl/sql/LabResultMigrator.groovy

```diff
@@ -0,0 +1,193 @@
+package org.pih.hivmigration.etl.sql
+
+class LabResultMigrator extends SqlMigrator {
+
+    @Override
+    def void migrate() {
+        executeMysql("Create staging table", '''
+            create table hivmigration_lab_results (
+              obs_id int PRIMARY KEY AUTO_INCREMENT,
```

Comment:
Note, each row in this table often represents an obs group with multiple obs within it.  I don't know if we need to explicitly set obs ids for anything, and can let them autogenerate.  We can keep uuids for all of the possible obs that each row might create, and this will allow us to set these and also use them to associate obs and obs groups together.

Reply:
OK, looks like you handle this below by using this obs_id as the group obs_id.  Might want to just rename this column to "obs_group_id" to be explicit about it.

Reply:
The obs group has an explicitly set `obs_id`, and its members have explicitly set `obs_group_id`s, which are that group's `obs_id`. It's exactly the way one creates obs normally, and a way of improving on it isn't obvious to me.

Reply:
Yeah, as I wrote above, maybe just renaming it to obs_group_id is more clearer